api/src/main/java/javax/jdo/Enhancer.java

```diff
@@ -85,6 +85,11 @@ public class Enhancer {
   /** Class file suffix */
   private static final String CLASS_FILE_SUFFIX = ".class"; // NOI18N
 
+  private static final String MSG_ENHANCER_CLASS_PATH = "MSG_EnhancerClassPath";
```

Comment:
DO we need to add `// NOI18N` ? Is that still a thing?

Reply:
Added NOI18N, also at a couple of other places.